src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -136,14 +136,15 @@ public void configureCommunication(@NonNull String host,
 
     private void configurePipes(URL backendUrl, Persona persona) {
         URL pipeUrl = Urls.getUrl(backendUrl, BackendPipes.Paths.ROOT);
-
+        URL alertReslveUrl = Urls.getUrl(pipeUrl, BackendPipes.Paths.ALERT_RESOLVE);
```

Comment:
Minor thing (and not blocker, of course).
Suggestion: alertResolveUrl  as a name of the variable ?